src/buildercore/bootstrap.py

```diff
@@ -162,6 +166,18 @@ def is_resourcing():
 
     stack_all_ec2_nodes(stackname, _setup_ec2_node, username=BOOTSTRAP_USER)
 
+def setup_terraform(stackname, context):
+    if not context.get('fastly'):
+        return
+    if context.get('fastly'):
+        if 'FASTLY_API_KEY' not in os.environ:
```

Comment:
this is configured out-of-band like AWS credentials, so should be mostly transparent for builder.

Reply:
`.prerequisites.py` candidate then?